fill-column-indicator.el

```diff
@@ -616,8 +620,11 @@ rough heuristic.)"
                    (dolist (win (fci-get-buffer-windows t))
                      (when (display-images-p (window-frame win))
                        (throw 'found-graphic (window-frame win))))))))
-    (setq fci-char-width (frame-char-width frame)
-          fci-char-height (frame-char-height frame))))
+    (setq fci-char-width (window-font-width)
+          fci-char-height (window-font-height)
+          fci-scaled-column (round (* (float (or fci-rule-column fill-column))
```

Comment:
`fci-get-frame-dimens` , that contains the `(or fci-rule-column fill-column)`,  is evaluated before `fci-column` is set (line 442).